recipes/xz_utils/all/conanfile.py

```diff
@@ -86,28 +80,29 @@ def _build_msvc(self):
                 use_env=False,
                 upgrade_project=False)
 
-    def _build_configure(self):
-        with tools.chdir(self._source_subfolder):
-            args = []
-            env_build = AutoToolsBuildEnvironment(self, win_bash=self._use_winbash)
-            args = ["--disable-doc"]
-            if self.settings.os != "Windows" and self.options.fPIC:
-                args.append("--with-pic")
-            if self.options.shared:
-                args.extend(["--disable-static", "--enable-shared"])
-            else:
-                args.extend(["--enable-static", "--disable-shared"])
-            if self.settings.build_type == "Debug":
-                args.append("--enable-debug")
-            env_build.configure(args=args, build=False)
-            env_build.make()
-            env_build.install()
+    def _configure_autotools(self):
+        if self._autotools:
+            return self._autotools
+        self._autotools = AutoToolsBuildEnvironment(self, win_bash=tools.os_info.is_windows)
+        args = ["--disable-doc"]
+        if self.settings.os != "Windows" and self.options.get_safe("fPIC", True):
```

Comment:
~since fPIC is deleted on windows is `get_safe` no enough?~ This flag enables so we need to limit that there's no else my mistake